tests/utils.py

```diff
@@ -108,7 +108,7 @@ def create_dataset(
 
     ds.tmin.encoding['_FillValue'] = np.float32(-9999999)
     ds.tmax.encoding['_FillValue'] = np.float32(-9999999)
-    ds.time.encoding['bounds'] = 'time_bounds'
+    ds.time.attrs['bounds'] = 'time_bounds'
```

Comment:
@benbovy / @andersy005 - want to draw your attention here. This change fixed the failing test on main.

Reply:
@jhamman, does this PR supersede PR #80?

Reply:
it does.

Reply:
My bad... I missed the comments in the PR body :)